npm/npm.go

```diff
@@ -188,6 +189,12 @@ func NewNetworkPolicyManager(clientset *kubernetes.Clientset, informerFactory in
 	iptMgr := iptm.NewIptablesManager()
 	iptMgr.UninitNpmChains()
 
+	log.Logf("Azure-NPM creating, cleaning existing IPSets")
+	destroyErr := ipsm.NewIpsetManager().Destroy()
```

Comment:
Wont this cleanup all the ipsets from that node? We should only clean the ones created by Azure-NPM. Also ignore the error as this is best effort to clean the old ipsets.

Reply:
As we discussed offline, NPM is not saving state any state while updating, to avoid any leak of ipsets, we will destroy everything. I logged a work item to add state fullness to this logic.

Reply:
Agree with Neha here, an ipset flush wipes out all ipsets, including ones NPM may have not created, this is where we need to iterate through all ipsets that contain azure-npm

Reply:
Added support to just delete azure-npm-(numerics) ipsets and lists